lib/vrl/web-playground/netlify.toml

```diff
@@ -2,18 +2,16 @@
 # all contexts unless otherwise overridden by more specific contexts.
 [build]
   # Directory to change to before starting a build.
-  # This is where we will look for package.json/.nvmrc/etc.
+  # This is where we will build the site in web-playground/public.
   # If not set, defaults to the root directory.
   base = "lib/vrl/web-playground/"
 
   # Directory that contains the deploy-ready HTML files and
-  # assets generated by the build. This is relative to the base
-  # directory if one has been set, or the root directory if
-  # a base has not been set. This sample publishes the directory
-  # located at the absolute path "root/project/build-output"
-
+  # assets generated by the build.
   publish = "public/"
 
   # Default build command.
-  command = "cargo install --version 0.10.3 wasm-pack && wasm-pack build --target web --out-dir public"
+  command = "cargo install --force --version 0.10.3 wasm-pack && wasm-pack build --target web --out-dir public"
```

Comment:
I wonder if there's a way to suppress that warning instead of ```--force```

Reply:
This seems ok. The downside is that it'll install it even if it already is installed. In some other places, we have guards that check if the given tool is already installed and skip if now. Like this:

https://github.com/vectordotdev/vector/blob/d21c420e33ccf603dd69bb9b20d9f4b278a53d6b/scripts/environment/prepare.sh#L8-L10